src/js/common/stores/PoliticianStore.js

```diff
@@ -136,6 +137,7 @@ class PoliticianStore extends ReduceStore {
     return positionsAboutPolitician[orgWeVoteId] || [];
   }
 
+  // Ask Dale about why this is needed - Zubin
```

Comment:
I've added this `getVoterCanEditThisPolitician` stub because we are planning to make it so the staff of politicians can update Politician information on WeVote, and the staff would be signed in with a WeVote voter account.